README.md

```diff
@@ -95,7 +95,7 @@ gulp help              # List the above commands.
 ## Usage
 
 ```bash
-node built/local/tsc.js hello.ts
+node <repo-root>/built/local/tsc.js hello.ts

```

Comment:
can you revert the changes here and in package.json? They're not related to the fix.

Reply:
Sorry, will do.